test/contractBridge/admin.js

```diff
@@ -21,10 +21,16 @@ contract('Bridge - [admin]', async accounts => {
     const initialRelayerThreshold = 2;
 
     const expectedBridgeAdmin = accounts[0];
+    const someone = accounts[4];
```

Comment:
may be more clear to name this `someRelayer`

Reply:
In this case this is just an arbitrary address, not even a relayer. But I think those tests would be even better if we check admin access against a relayer instead of any address. I'll update it.